### PR TITLE
[DOCS] Update GX Cloud Docs to Reflect UI Updates

### PR DESCRIPTION
**Pull request by kwcanuck**
In [this Slack conversation](https://greatexpectationslabs.slack.com/archives/C06TBEX44VC/p1712685458728199), @annabarr indicated that "a release (was pushed) this morning for pre-loading data asset schemas." As a result of this change, the GX Cloud documentation needed updating. Specifically, _[Manage Data Assets](https://docs.greatexpectations.io/docs/cloud/data_assets/manage_data_assets)_. This PR implements the necessary changes

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (Shinnnyshinshin) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: 🚀

**Review by Shinnnyshinshin — APPROVED** [review]
LGTM. Tagging @annabarr in case any of the wording should be changed.

**Review by Rachel-Reverie — APPROVED** [review]
LGTM

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (annabarr) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Since the overview page is automatically selected the first time a Data Asset is opened, should this line go above in step 1? Recommend "When you select a new Data Asset, the schema is automatically fetched."

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (annabarr) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Clarification: the button says "Profile Data" before additional metrics are calculated the first time. After that the button says "Refresh" as usual

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @annabarr, the text you identified is referred to as a step result. Step results appear _after_ the action that caused them. Users will see the Data Asset metrics being generated, and the statement will have context and meaning. If the information is presented in the introductory statement, it lacks context and could cause confusion.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @annabarr, I missed this. Thank you. I created a new Data Asset to test the functionality. This isn't intuitive. Having different text for similar actions makes documentation challenging.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (annabarr) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Works for me. Thoughts on language? We could be more specific and say the schema is fetched automatically, and not metrics

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @annabarr, if we're changing the terminology from _metrics_ to _schema_, it would probably be best to have a wider conversation. We've been using _metrics_  until this change. Do you want to initiate a conversation on the Slack Product channel? I'll update the docs based on the results of that conversation.

Comment (annabarr) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: We used to call both the schema fetch and the metrics fetch "metrics" because we queried for them at the same time. They are distinct operations, and this work was to fetch the schema automatically and fetch the metrics on demand (because they are more expensive to compute). We can continue to call the schema fetch a subset of "metrics", but it's not exactly right since we are not calculating any numbers.

@kwcanuck If you think it's more confusing to add a new term I am fine with keeping it as is!

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @annabarr will users understand the distinction between metrics and schema? Will they understand they are distinct operations? Will users understand that they are fetching the schema when selecting Profile Data? If the answer to these questions is "no," I recommend keeping the current terminology. Your call.

Comment (annabarr) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Maybe not. What about "When you select a new Data Asset, columns and data types are automatically fetched."

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @annabarr, with the text you've suggested, I immediately think, "what columns and data types are fetched?" and "what's a data type?" I think a simple, straightforward statement is best. Maybe:

 _When you select a new Data Asset, schema data is automatically fetched._ 

I'll let _schema_ slide!

Comment (annabarr) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Haha ok sounds good to me!

**Review by annabarr — APPROVED** [review]
Approved pending final edit!

**Pull request merged**
